forge-game/src/main/java/forge/game/trigger/TriggerDrawn.java

```diff
@@ -87,6 +87,13 @@ public final boolean performTest(final Map<AbilityKey, Object> runParams) {
             }
         }
 
+        if (hasParam("FirstCardInDrawStep")) {
+            final Player p = ((Player)runParams.get(AbilityKey.Player));
+            if (!game.getPhaseHandler().is(PhaseType.DRAW, p) || p.numDrawnThisDrawStep() > 1) {
```

Comment:
Can you test this with _Sphinx of the Second Sun_?

I think PhaseHandler might not reset numDrawnThisStep in that case yet

Reply:
GOOD CALL – it wasn't working... Xyris wasn't interacting with it properly either.

Think I've got it fixed, though!